drainer/util.go

```diff
@@ -58,12 +58,12 @@ func (g *taskGroup) start(name string, f func(), noPanic bool) {
 				if err := recover(); err != nil {
 					log.Error("Recovered from panic",
 						zap.Reflect("err", err),
-						zap.Stack("stack"),
+						zap.Stack("real stack"),
 						fName,
 					)
 				}
 			}
-			log.Info("Exit", fName)
+			log.Error("Exit", fName)
```

Comment:
If `f()` returns successfully, this line is also reached because it's in `defer`, so it's not always an error?